frontend/src/app/plans/page.tsx

```diff
@@ -0,0 +1,51 @@
+"use client";
+
+import { atom, useAtom } from "jotai";
+import { PlusIcon } from "lucide-react";
+import { useRouter } from "next/navigation";
+
+import { planFamily } from "@/atoms/planFamily";
+import { plansIds } from "@/atoms/plansIds";
+import { Plan } from "@/components/Plan";
+
+const plansAtom = atom(
+  (get) => get(plansIds).map((id) => get(planFamily(id))),
+  (get, set, values: Array<{ id: string }>) => {
+    set(plansIds, values);
+  },
+);
+const Plans = () => {
+  const [plans, setPlans] = useAtom(plansAtom);
+  const router = useRouter();
+  const uuid = crypto.randomUUID();
```

Comment:
można to przenieść do funkcji addNewPlan, żeby przy każdym rerenderze uuid się nie generował